kotlin-css/src/main/kotlin/kotlinx/css/StyleEnums.kt

```diff
@@ -89,237 +93,408 @@ enum class Clear {
-        val lightslategrey = Color("#778899")
-        val lightsteelblue = Color("#b0c4de")
-        val lightyellow = Color("#ffffe0")
-        val lime = Color("#00ff00")
-        val limegreen = Color("#32cd32")
-        val linen = Color("#faf0e6")
-        val magenta = Color("#ff00ff")
-        val maroon = Color("#800000")
-        val mediumaquamarine = Color("#66cdaa")
-        val mediumblue = Color("#0000cd")
-        val mediumorchid = Color("#ba55d3")
-        val mediumpurple = Color("#9370db")
-        val mediumseagreen = Color("#3cb371")
-        val mediumslateblue = Color("#7b68ee")
-        val mediumspringgreen = Color("#00fa9a")
-        val mediumturquoise = Color("#48d1cc")
-        val mediumvioletred = Color("#c71585")
-        val midnightblue = Color("#191970")
-        val mintcream = Color("#f5fffa")
-        val mistyrose = Color("#ffe4e1")
-        val moccasin = Color("#ffe4b5")
-        val navajowhite = Color("#ffdead")
-        val navy = Color("#000080")
-        val oldlace = Color("#fdf5e6")
-        val olive = Color("#808000")
-        val olivedrab = Color("#6b8e23")
-        val orange = Color("#ffa500")
-        val orangered = Color("#ff4500")
-        val orchid = Color("#da70d6")
-        val palegoldenrod = Color("#eee8aa")
-        val palegreen = Color("#98fb98")
-        val paleturquoise = Color("#afeeee")
-        val palevioletred = Color("#db7093")
-        val papayawhip = Color("#ffefd5")
-        val peachpuff = Color("#ffdab9")
-        val peru = Color("#cd853f")
-        val pink = Color("#ffc0cb")
-        val plum = Color("#dda0dd")
-        val powderblue = Color("#b0e0e6")
-        val purple = Color("#800080")
-        val red = Color("#ff0000")
-        val rosybrown = Color("#bc8f8f")
-        val royalblue = Color("#4169e1")
-        val saddlebrown = Color("#8b4513")
-        val salmon = Color("#fa8072")
-        val sandybrown = Color("#f4a460")
-        val seagreen = Color("#2e8b57")
-        val seashell = Color("#fff5ee")
-        val sienna = Color("#a0522d")
-        val silver = Color("#c0c0c0")
-        val skyblue = Color("#87ceeb")
-        val slateblue = Color("#6a5acd")
-        val slategray = Color("#708090")
-        val slategrey = Color("#708090")
-        val snow = Color("#fffafa")
-        val springgreen = Color("#00ff7f")
-        val steelblue = Color("#4682b4")
-        val tan = Color("#d2b48c")
-        val teal = Color("#008080")
-        val thistle = Color("#d8bfd8")
-        val tomato = Color("#ff6347")
-        val turquoise = Color("#40e0d0")
-        val violet = Color("#ee82ee")
-        val wheat = Color("#f5deb3")
-        val white = Color("#ffffff")
-        val whitesmoke = Color("#f5f5f5")
-        val yellow = Color("#ffff00")
-        val yellowgreen = Color("#9acd32")
+        val currentColor = Color("currentColor")
+
+        // W3C predefined HTML colors (140), see the referenced specification above.
```

Comment:
I'm not sure about capitalization at all. The color names in the spec are in lowercase.